api/src/main/resources/workbench-api.yaml

```diff
@@ -6986,6 +6989,15 @@ components:
           type: boolean
           default: false
           description: If true, allow for user's initial credits to expire.
+        initialCreditsValidityPeriodDays:
+          type: integer
+          description: The number of days that a user's initial credits are valid for.
+        initialCreditsExtensionPeriodDays:
+          type: integer
+          description: The number of days that a user's initial credits can be extended for.
+        initialCreditsExpirationWarningDays:
```

Comment:
for my knowledge, when do these 3 values get populated? are they user specific?

Reply:
These are config values, so they are not user specific. They are loaded whenever the /config endpoint is called.